enumeration-set/extension/gx-name-qualifiers.yaml

```diff
@@ -0,0 +1,26 @@
+%YAML 1.2
+---
+lang: en-US
+
+type: enumeration set
+
+uri: https://tychonievich.github.io/gx-g7-names#qualifiers
```

Comment:
Who should the `contact` be for this YAML file?  The GEDCOM X spec maintainers? Luther personally?